jardin/config.py

```diff
@@ -25,4 +26,8 @@ def init():
     logger = logging.getLogger('jardin')
     logger.setLevel(LOG_LEVEL)
     
-    setattr(this, 'INITIALIZED', True)
\ No newline at end of file
+    setattr(this, 'INITIALIZED', True)
+
+
+def reset_session():
```

Comment:
i am not sure if you need this defined in config or did you mean to have this in jardin/__init__.py ?

Reply:
I can move it outside config